Examples/frame-sheet-name-test.lisp

```diff
@@ -0,0 +1,38 @@
+(cl:in-package #:clim-demo)
+
+(define-application-frame frame-sheet-name-test ()
+  ()
```

Comment:
when there is no need for menubar, provide `(:menu-bar nil)` otherwise McCLIM will add an empty menu-bar visible as 4-pixel line below title-bar